eng/validation/build-arcadewithrepo.ps1

```diff
@@ -195,52 +210,105 @@ function Git-Command($repoName) {
     }
 }
 
-## If able to retrieve a build, get the SHA that it was built from
-$sha = Get-LastKnownGoodBuildSha -buildDefinitionId $buildDefinitionId
-
-## Clone the repo from git
-GitHub-Clone $targetRepoName
-
-## Create a branch from the repo with the given SHA. 
+## Global Variables
+$githubUri = Get-Github-RepoAuthUri $githubRepoName
+$azdoUri = Get-AzDO-RepoAuthUri $azdoRepoName
+$remoteName = ($azdoOrg + "-" + $azdoRepoName)
 $targetBranch = "dev/" + $githubUser + "/arcade-" + $arcadeSdkVersion
-$githubUri = Get-Github-RepoAuthUri $targetRepoName
-
 $darcBranchName = "refs/heads/" + $targetBranch
-$darcRepoName = "https://github.com/${githubOrg}/${targetRepoName}"
+$darcGitHubRepoName = "https://github.com/${githubOrg}/${githubRepoName}"
+$darcAzDORepoName = "https://${azdoOrg}.visualstudio.com/${azdoProject}/_git/${azdoRepoName}"
```

Comment:
We should use the dev.azure.com name rather than the visualstudio url for this.

Reply:
Done.